hypervisor/Cargo.toml

```diff
@@ -14,6 +14,7 @@ tdx = []
 [dependencies]
 anyhow = "1.0.86"
 byteorder = "1.5.0"
+concat-idents = "1.1.5"
```

Comment:
This crate seems to be not maintained. It was updated over a year ago with opening issues and PRs for over several months. Can you please find alternatives? Thanks.

Reply:
This API is part of nightly compiler already https://doc.rust-lang.org/std/macro.concat_idents.html. I don't know if there are any crates which provide a similar functionality.